### ISSUE TITLE
Audit application heading levels

### ISSUE THREAD

**Issue by katypies** [cisagov/manage.get.gov#4487]
### Issue description

In #3846, we're fixing an instance where there is an H1 on a page, and H3s, but no H2 headings, which is an accessibility violation (https://www.w3.org/WAI/tutorials/page-structure/headings/). We should make sure we're not doing this in other places in the application, and assess if we can detect this as part of our development workflows to prevent it going forward. 

### Acceptance criteria

- [ ] Determine if we can programmatically detect skipped heading levels
    - [ ] Create script or test to look for skipped headings
    - [ ] Implement test or script
- [ ] If this cannot be done programmatically for new PRs, conduct an audit of application pages to ensure that no heading levels are skipped

### Links to other issues

- 🔄 Relates to #3846
